SofaScoreAcademy/SofaScoreAcademy/Views/MatchView.swift

```diff
@@ -0,0 +1,223 @@
+            $0.top.equalTo(matchCellView.snp.top).offset(10)
+            $0.width.equalTo(32.0)
+            $0.height.equalTo(16.0)
+            
+        }
+        
+        teamTwoScoreLabel.snp.makeConstraints {
+            
+            $0.leading.equalTo(teamOneScoreLabel)
+            $0.top.equalTo(teamOneScoreLabel.snp.bottom).offset(4)
+            $0.width.equalTo(32.0)
+            $0.height.equalTo(16.0)
+            
+        }
+        
+        teamOneImageView.snp.makeConstraints {
+            
+            $0.leading.equalTo(matchCellView.snp.leading).offset(80)
+            $0.top.equalTo(matchCellView.snp.top).offset(10)
+            $0.width.height.equalTo(16.0)
+            
+        }
+        
+        teamTwoImageView.snp.makeConstraints {
+            
+            $0.leading.equalTo(teamOneImageView)
+            $0.top.equalTo(teamOneScoreLabel.snp.bottom).offset(4)
+            $0.width.height.equalTo(16.0)
+            
+        }
+        
+        matchTimeLabel.snp.makeConstraints {
+            
+            $0.leading.equalTo(matchCellView.snp.leading).offset(4)
+            $0.top.equalTo(matchCellView.snp.top).offset(10)
+            $0.width.equalTo(56.0)
+            $0.height.equalTo(16.0)
+            
+        }
+        
+        matchStatusLabel.snp.makeConstraints {
+            
+            $0.leading.equalTo(matchTimeLabel)
+            $0.top.equalTo(teamOneScoreLabel.snp.bottom).offset(4)
+            $0.width.equalTo(56.0)
+            $0.height.equalTo(16.0)
+            
+        }
+        
+        matchCellverticalDivider.snp.makeConstraints {
+            
+            $0.leading.equalTo(matchCellView.snp.leading).offset(63)
+            $0.top.equalTo(matchCellView.snp.top).offset(8)
+            $0.width.equalTo(1.0)
+            $0.height.equalTo(40.0)
+            
+        }
+    }
+    
+    func configure(with event: Event) {
+        teamOneNameLabel.text = event.teamOneName
+        teamTwoNameLabel.text = event.teamTwoName
+        matchTimeLabel.text = event.formattedStartTime
+        
+        let teamOneScore = event.teamOneScore ?? 0
+        let teamTwoScore = event.teamTwoScore ?? 0
+        
+        teamOneScoreLabel.text = teamOneScore.description
+        teamTwoScoreLabel.text = teamTwoScore.description
+        updateStatus(for: event.matchStatus, with: event.matchTime, teamOneScore: teamOneScore, teamTwoScore: teamTwoScore)
+    }
+
+    private func updateStatus(for status: MatchStatus, with time: Int?, teamOneScore: Int, teamTwoScore: Int) {
+        let winnerCode: WinnerCode? = {
+            if status == .isFinished {
+                if teamOneScore > teamTwoScore {
+                    return .home
+                } else if teamOneScore < teamTwoScore {
+                    return .away
+                }
+            }
+            return nil
+        }()
+
+        let colors = MatchView.getTeamNameColors(for: status, winnerCode: winnerCode)
```

Comment:
i kad nece bit static moc ces ovako samo
```suggestion
        let colors = getTeamNameColors(for: status, winnerCode: winnerCode)
```